scripts/oo-request/disputePrice.js

```diff
@@ -0,0 +1,79 @@
+const hre = require("hardhat");
+const ethers = hre.ethers;
+const createOOContractInstance = require("../helpers/createOOContractInstance");
+const createERC20ContractInstance = require("../helpers/createERC20ContractInstance");
+
+const ethersLib = require("ethers");
+const web3 = require("web3");
+const toWei = require("../helpers/toWei");
+
+const MAX_INT = ethers.constants.MaxUint256;
+
+/**
+ * @notice Disputes a price value for an existing price request with an active proposal.
+ * @param requester sender of the initial price request.
+ * @param identifier price identifier to identify the existing request.
+ * @param timestamp timestamp to identify the existing request.
+ * @param ancillaryData ancillary data of the price being requested.
+ * @return totalBond the amount that's pulled from the disputer's wallet as a bond. The bond will be returned to
+ * the disputer once settled if the dispute was valid (the proposal was incorrect).
+ */
+/*
+     function disputePrice(
+      address requester,
+      bytes32 identifier,
+      uint256 timestamp,
+      bytes memory ancillaryData
+  ) external virtual returns (uint256 totalBond);
+*/
+
+async function main() {
+  const foundationAccount = "0x7a3a1c2de64f20eb5e916f40d11b01c441b2a8dc";
+  const ooAddress = "0xc43767f4592df265b4a9f1a398b97ff24f38c6a6";
+  const hardhatAccount0 = "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266";
```

Comment:
since this script is becoming multichain we shold probably have a constnats file with an object indexed by chain to get all these. the other option is to use the `contracts-node` library to divine these values using `await getAddress('name',network)`